### DISCUSSION TITLE
Different sql for each table

### DISCUSSION THREAD

**Discussion by yhackup** [MySqlBackupNET/MySqlBackup.Net#112] [Q&A] [answered]
Hello, can I get different SQL output for each table? I searched in ExportInfo but couldn't find it.

**Comment by adriancs2** [comment, marked as answer]
You can obtain a list of tables, then perform a loop to export each table as individual backup file.

A minute ago, I have published a simple wiki page for this idea. Here's the link to the page:

https://github.com/MySqlBackupNET/MySqlBackup.Net/wiki/Export-Each-Table-As-Separate-SQL-Backup-File

**Reply by yhackup** [reply]
thank you.
